skimage/feature/blob.py

```diff
@@ -240,13 +240,20 @@ def blob_dog(image, min_sigma=1, max_sigma=50, sigma_ratio=1.6, threshold=2.0,
     sigma_ratio : float, optional
         The ratio between the standard deviation of Gaussian Kernels used for
         computing the Difference of Gaussians
-    threshold : float, optional.
+    threshold : float or None, optional.
         The absolute lower bound for scale space maxima. Local maxima smaller
-        than thresh are ignored. Reduce this to detect blobs with less
-        intensities.
+        than `threshold` are ignored. Reduce this to detect blobs with lower
+        intensities. If `threshold_rel` is also specified, whichever threshold
+        is larger will be used. If None, `threshold_rel` is used instead.
     overlap : float, optional
         A value between 0 and 1. If the area of two blobs overlaps by a
         fraction greater than `threshold`, the smaller blob is eliminated.
+    threshold_rel : float or None, optional
+        Minimum intensity of peaks, calculated as
+        ``max(dog_space) * threshold_rel``. Where ``dog_space`` refers to the
```

Comment:
```suggestion
        ``max(dog_space) * threshold_rel``, where ``dog_space`` refers to the
```
(no big deal)

Reply:
PS: Should we write 'image_cube' instead of 'dog_space' to match the source code? Or is just formal (~ pseudo-code), sensible variable naming for the docstring?